photo/mutations.py

```diff
@@ -64,15 +64,15 @@ def create_picture(self, input: PictureInput, picture: Upload) -> PictureType:
         picture.save(image_bytes, format="JPEG")
         image_bytes.seek(0)
         image_file = SimpleUploadedFile(
-            "sample.jpg", image_bytes.getvalue(), content_type="image/jpeg"
+            input.name, image_bytes.getvalue(), content_type="image/jpeg"
         )
 
         user = User.objects.get(email=input.user)
 
-        newPicture = Picture(user=user, file=image_file)
-        newPicture.save()
+        picture = Picture(user=user, file=image_file)
```

Comment:
maybe change this variable name since we already have a picture in the arguments

Reply:
Changed